ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-cert-manager-1.18.yaml

```diff
@@ -0,0 +1,147 @@
+base_images:
+  base-rhel9:
+    name: "4.20"
+    namespace: ocp
+    tag: base-rhel9
+  cli-operator-sdk:
+    name: cli-operator-sdk
+    namespace: ocp
+    tag: v1.39.2
+  ocp_builder_rhel-9-golang-1.24-openshift-4.20:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.24-openshift-4.20
+binary_build_commands: make build --warn-undefined-variables
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: images/ci/Dockerfile
+  from: base-rhel9
+  inputs:
+    ocp_builder_rhel-9-golang-1.24-openshift-4.20:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.24-openshift-4.20
+  to: cert-manager-operator
+- dockerfile_path: images/ci/operand.Dockerfile
+  from: base-rhel9
+  inputs:
+    ocp_builder_rhel-9-golang-1.24-openshift-4.20:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.24-openshift-4.20
+  to: cert-manager
+operator:
+  bundles:
+  - as: cert-manager-operator-bundle
+    dockerfile_path: bundle.Dockerfile
+    skip_building_index: true
+  substitutions:
+  - pullspec: openshift.io/cert-manager-operator:latest
+    with: pipeline:cert-manager-operator
+  - pullspec: quay.io/jetstack/cert-manager-controller:latest
+    with: pipeline:cert-manager
+  - pullspec: quay.io/jetstack/cert-manager-webhook:latest
+    with: pipeline:cert-manager
+  - pullspec: quay.io/jetstack/cert-manager-cainjector:latest
+    with: pipeline:cert-manager
```

Comment:
```suggestion
  - pullspec: openshift.io/cert-manager-operator:.*
    with: pipeline:cert-manager-operator
  - pullspec: quay.io/jetstack/cert-manager-controller:.*
    with: pipeline:cert-manager
  - pullspec: quay.io/jetstack/cert-manager-webhook:.*
    with: pipeline:cert-manager
  - pullspec: quay.io/jetstack/cert-manager-cainjector:.*
    with: pipeline:cert-manager
```

Reply:
Wondering why is this change required and won't it break the exisiting substitution : https://github.com/openshift/cert-manager-operator/blob/97ab881d6273393a3aa2acccadddcb3d502ec9a8/bundle/manifests/cert-manager-operator.clusterserviceversion.yaml#L659-L663 ?

Reply:
I am not sure about this